ringbuf/reader_test.go

```diff
@@ -269,12 +288,18 @@ func TestReaderNoWakeup(t *testing.T) {
 	}
 	defer rd.Close()
 
+	qt.Assert(t, qt.Equals(rd.AvailableData(), uint64(0)))
+
 	ret, _, err := prog.Test(internal.EmptyBPFContext)
 	testutils.SkipIfNotSupported(t, err)
 	if err != nil {
 		t.Fatal(err)
 	}
 
+	availableData := 3 * 16 // 8 bytes + 8 bytes for the ringbuf header
+
+	qt.Assert(t, qt.Equals(rd.AvailableData(), uint64(availableData)))
```

Comment:
Could you do the following instead, for all places where you compare with a constant?
```suggestion
	qt.Assert(t, qt.Equals(rd.AvailableData(), 3* 16))
```

That way we get the constant in the output of the test failure.